README.md

```diff
@@ -1,4 +1,39 @@
 # bootstrap_go
 Lightweight bootstrap nodes for Golem network. Basic implementation in Go.
 
+# installation
+
+## prerequisites 
+- install `go` tools https://golang.org/doc/install  
+- create go workspace `mkdir ~/go` https://github.com/golang/go/wiki/SettingGOPATH
+
+## get code and deps
+
+This will get the sources into default `GOPATH` which is `~/go`.
+
+```
+go get github.com/golemfactory/bootstrap_go
+cd ~/go/src/github.com/golemfactory/bootstrap_go/
+# get dependencies and test dependencies
+go get -v -t ./...
```

Comment:
`-v` is for verbose output which isn't usually really needed

Reply:
ok